src/DotNet.Status.Web/tests/DotNet.Status.Web.Tests/TelemetryControllerTests.cs

```diff
@@ -0,0 +1,75 @@
+using Xunit;
+using DotNet.Status.Web.Controllers;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Moq;
+using Microsoft.AspNetCore.Mvc;
+using Kusto.Ingest;
+using System.Threading.Tasks;
+
+namespace DotNet.Status.Web.Tests
+{
+    public class TelemetryControllerTests
+    {
+        [Fact]
+        public void TestArcadeValidationTelemetryCollection()
```

Comment:
I'm... suprized this tests works.  It's going to attempt to create an injection client from "fakekustoconnectionstring", and that _has_ to throw... so I would expect this test to throw.

You'll probably need to DI the a whole KustoClientProvider (and hook that up in Startup). Then your tests can hook up a fake one. :-)

Then it can, bonus, even assert that something got written to kusto.